cmd/performance-profile-creator/cmd/root.go

```diff
@@ -34,11 +34,20 @@ import (
 	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
 )
 
+const (
+	singleNumaNodeTopologyManagerPolicy = "single-numa-node"
+)
+
+var (
+	validTMPolicyValues = []string{"single-numa-node", "best-effort", "restricted", "none", ""}
```

Comment:
Why do we need 2 NOP options, "none" and  "" ?
Can we even have a none TM policy? I wasn't aware of it.

Reply:
We don't need "" anymore. In the previous implementation, in case the user wasn't specifying any value it would default to "" but as we default to "restricted" that is no longer relevant.
Yes "none" is one the TM policies. Please refer [here](https://kubernetes.io/docs/tasks/administer-cluster/topology-manager/#topology-manager-policies).

Reply:
can't we just import the k8s/ocp names? duplicating data (and validation) always carries some risk.